ssr-express/spec/lib/AmpSsrUrlSchemeSpec.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const AmpSsrUrlScheme = require('../../lib/AmpSsrUrlScheme');
+
+describe('AmpSsrUrlScheme', () => {
+  const ampSsrUrlScheme = new AmpSsrUrlScheme('amp');
+
+  describe('isAmpUrl', () => {
```

Comment:
Can you add a test for `ampSsrUrlScheme.isAmpUrl('/page?amp');`

Reply:
Added